input/rmatch.in

```diff
@@ -1,94 +1,94 @@
 &RMATCH_INPUT
-    galsol_filename = "gal_solution.dat"
-    galsol_filename_cut = "gal_solution_cut.dat"
-    deltabin_filename = "delta_gw.dat"
+    galsol_filename = "gal_solution.dat"          ! File containing Galerkin solutions
+    galsol_filename_cut = "gal_solution_cut.dat"  ! File containing resistive/cut Galerkin solutions
+    deltabin_filename = "delta_gw.dat"            ! File containing delta matrix from RDCON
 
-    match_flag=t
-    scan_flag=f
-    qscan_flag=f
-    sol_flag =f
-    deflate=f
-    nroot=5
-    itermax=5000
-    relax_fac=0.5
-    msing=20
-    initguess = ( 0,10)
-    eta=8e-8,8e-8,8e-8,8e-8                    ! Resistivity at each rational surface, starting from the core
-    massden=3.3e-07,3.3e-07,3.3e-07,3.3e-07             ! Mass density (kg/m^3) at each rational surface, starting from the core. 3.3e-7 is ni=1e20 m^-3.
-    scan_x1 =  1e-9
-    scan_x0 = 1.90E-10
+    ! Scans, solves, and matching options. Not used with RPEC.
+    match_flag=t         ! Compute inner region matching data; not used with RPEC
+    scan_flag=f          ! Scan over eta; not used with RPEC
+    qscan_flag=f         ! Scan over eigenvalue Q; not used with RPEC
+    sol_flag =f          ! Newton solve for single eigenvalue; not used with RPEC
+    deflate=f            ! Divides Delta determinant by PRODUCT(guess-oldroots)
+    nroot=5              ! Number of eigenvalues to find
+    itermax=5000         ! Maximum number of iterations in Newton solve
+    relax_fac=0.5        ! Numerical relaxation factor in Newton solve
+    msing=4              ! Number of singular surfaces to include in matching
+    initguess = ( 0,10)  ! Initial guess for eigenvalues in Newton solve
+    scan_x0 = 1.90E-10   ! Start of scan range if doing any scan
+    scan_x1 =  1e-9      ! End of scan range if doing any scan
+    scan_nstep = 50      ! Number of steps in scan if doing any scan
+    qscan_ising=2        ! Which singular surface to scan Q over
+    deltar_flag=t        ! Run and record Galerkin GGJ model in Fourier space
+    deltac_flag=t        ! Run and record Galerkin GGJ model in configuration space
```

Comment:
For deltac consider: 'solve inner layer equations using cubic hermite finite element method'